lib/pitchfork/http_server.rb

```diff
@@ -480,7 +480,7 @@ def master_sleep(sec)
 
     def awaken_master
       return if $$ != @master_pid
```

Comment:
What about these two?

Reply:
I can add another commit to the PR.

Reply:
> Does this warrant a mention in the Changelog?

I don't think so. This is all internal stuff, and there's no reason to monkey patch Pitchfork, I made sure to provide the proper hooks that Unicorn lacked to do things like OOB GC etc.

Any mention of `master` is me being a bit lazy and justifying it by telling myself it would make it easier to upstream patches back to Unicorn if necessary, but I think that ship has sailed.